build/version.js

```diff
@@ -0,0 +1,21 @@
+import {readFileSync, writeFileSync} from 'node:fs';
+
+// TODO: Replace this with a regular import when ESLint adds support for import assertions.
+// See: https://rollupjs.org/guide/en/#importing-packagejson
+const pkg = JSON.parse(readFileSync(new URL('../package.json', import.meta.url)));
+
+const fileContent = readFileSync(new URL('../src/Leaflet.js', import.meta.url), 'utf8');
+
+const newContent = fileContent.replace(
+	/(\/\/ !!! NEXT LINE IS AUTO-GENERATED VIA `NPM VERSION` !!!\s+export const version = ')[^']+(')/,
```

Comment:
Do we want to also support `"` (in case we ever switch quoting style)?

```suggestion
	/(\/\/ !!! NEXT LINE IS AUTO-GENERATED VIA `NPM VERSION` !!!\s+export const version = ['"])[^'"]+(['"])/,
```

Reply:
Thanks for the suggestion but I think we keep it simple. If the replacement isn't working then the command will fail.